### PR TITLE
Created all neccesery modules, classes and methods

### PR DESCRIPTION
**Pull request by shafran-andriy**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    sum_of_ratings = 0
+    for player in players:
```
Comment (vasylhnatiuk) [inline app/main.py:8]: use list comprehension here

File: app/players/player.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
```
Comment (vasylhnatiuk) [inline app/players/player.py:9]:
```suggestion
    def get_rating(self) -> int:
```

File: app/players/player.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        pass
+
+    @abstractmethod
+    def player_info(self) -> None:
```
Comment (vasylhnatiuk) [inline app/players/player.py:13]:
```suggestion
    def player_info(self) -> str:
```

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (shafran-andriy) [reply app/main.py:8]: fixed

File: app/players/player.py (hunk 2 above)
Comment (shafran-andriy) [reply app/players/player.py:9]: fixed

File: app/players/player.py (hunk 3 above)
Comment (shafran-andriy) [reply app/players/player.py:13]: fixed

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**